packages/@aws-cdk-containers/ecs-service-extensions/lib/extensions/scale-on-cpu-utilization.ts

```diff
@@ -126,6 +134,9 @@ export class ScaleOnCpuUtilization extends ServiceExtension {
   // This hook utilizes the resulting service construct
   // once it is created.
   public useService(service: ecs.Ec2Service | ecs.FargateService) {
+    if (this.parentService.scalableTaskCount) {
+      throw Error('Cannot specify \'minTaskCount\' or \'maxTaskCount\' in the Service construct and also provide a  \'ScaleOnCpuUtilization\' extension. \'ScaleOnCpuUtilization\' is deprecated. Please only provide \'minTaskCount\' and \'maxTaskCount\'.');
```

Comment:
Just realized we can make this error a little bit shorter and clearer by referring to the property name that contains `minTaskCount` and `maxTaskCount`. 

```suggestion
      throw Error('Cannot specify \'autoScaleTaskCount\' in the Service construct and also provide a  \'ScaleOnCpuUtilization\' extension. \'ScaleOnCpuUtilization\' is deprecated. Please only provide \'autoScaleTaskCount\'.');
```